src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -819,14 +807,177 @@ bool AutomationInstrumentClipView::renderSidebar(uint32_t whichRows, uint8_t ima
 	return instrumentClipView.renderSidebar(whichRows, image, occupancyMask);
 }
 
```

Comment:
new function created to render information on OLED and 7Seg displays

On Automation Overview:
- on OLED it renders "Automation Overview" (or "Can't Automate CV" if you're on a CV clip)
- on 7Seg it renders AUTO (or CANT if you're on a CV clip)